### ISSUE TITLE
Download Images

### ISSUE THREAD

**Issue by fnando1995** [MissiaL/hikvision-client#25]
@MissiaL Is there a way to increase the resolution of the image? and also a way to declare the exact second to download the image? (I know that right now you run the code and it downloads the live-view image at 704x576, I could use a 1920x1800 image, live-view, or (better) an expecific time-selected image )

Thanks in advance for any help.

**Comment by igorvukotic** [comment]
Resolution is depends on resolution what is setup-ed on camera.

**Comment by fnando1995** [comment]
I've tested this in different cameras, and I can relate this: 

if the DVR/NVR if configured to high resolution, the camera will download a 704x576 jpg image. In the other hand, if the net device is configured to a resolution lower than 704x576, then the downloaded image will have that resolution lower reso. 

example:
dvr1 saves images at 1920x1800, downloaded image's size will be: 704x576
dvr2 saves images at 640x480, downloaded image's size will be: 640x480

In conclusion, maybe there is a parameter which is reducing the resolution of the image.

**Comment by AndrewMarines** [comment]
@fnando1995 Did you find a way to do that? I'm currently stucked and probably my [issue ](https://github.com/MissiaL/hikvision-client/issues/32)won't be resolved cause the creator doesn't have a HIKVision camera anymore

**Comment by fnando1995** [comment]
@AndrewMarines Saddly no. If you get the problem solved, please let me know.

**Comment by AndrewMarines** [comment]
> @AndrewMarines Saddly no. If you get the problem solved, please let me know.

Even if it's bad, we just need to understand where to append` ?videoResolutionWidth=1920&videoResolutionHeight=1080` to the url. This fixes the problem but idk where to append it in this code

**Comment by fnando1995** [comment]
@AndrewMarines I make this [repo](https://github.com/fnando1995/hiktool.git) for wrapping the SDK. This works on normal PCs (x64 machines), but for others you should rebuild the sdk, and in cases like arm, there ir no way to rebuild.

**Comment by MissiaL** [comment]
Hi! I'm trying to help you
Try adding a field to your request
```
params={'videoResolutionWidth':'1920', 'videoResolutionHeight':'1080'}
```


For example
```
response = client.Streaming.channels[102].picture(method='get', type='opaque_data', params={'videoResolutionWidth':'1920', 'videoResolutionHeight':'1080'})
```

I'm not sure but maybe this should work

**Comment by AndrewMarines** [comment]
> 

@fnando1995 could you try please? I gotta wait at least a week to see if this works fine while standing next to  the customer

**Comment by fnando1995** [comment]
@AndrewMarines Just did, and worked. Thanks @MissiaL . I must say I would appreciated it more one year ago XD

**Comment by AndrewMarines** [comment]
Love you guys! Have a good day

**Comment by jak83A** [comment]
The solution lies in the number of the stream which you want to use for the snapshot...

100 gives me 3840x2160 | 101 gives 3840x2160 | 102 gives 1280x720

`response = cam1.Streaming.channels[100].picture(method='get', type='opaque_data')`
